.github/workflows/deploy-ecs.yaml

```diff
@@ -51,11 +46,40 @@ jobs:
           echo "Deploying image: $IMAGE_URI"
           echo "ECS Service: $ECS_SERVICE"
           echo "ECS Cluster: $ECS_CLUSTER"
+          echo "Task Definition: $TASK_DEFINITION"
+          echo "Container Name: $CONTAINER_NAME"
           
-          # Update the ECS service with the new image
+          # Get the current task definition
+          TASK_DEFINITION_ARN=$(aws ecs describe-task-definition \
+            --task-definition $TASK_DEFINITION \
+            --query 'taskDefinition.taskDefinitionArn' \
+            --output text)
+          
+          echo "Current task definition ARN: $TASK_DEFINITION_ARN"
+          
+          # Download the current task definition
+          aws ecs describe-task-definition \
```

Comment:
every call to `aws ... ` is heavy and can fail. don't call it twice if you already called it a sec ago on line 53

Reply:
You can first save everything to `task-definition.json` and get the ARN from the file.